packages/app-form-builder/src/admin/components/FormEditor/Draggable.tsx

```diff
@@ -17,8 +16,15 @@ interface BeginDragProps {
         row: number;
         index?: number;
     };
-    formStep?: FbFormStep;
     name?: string;
+    id?: string;
+    /*
+        "container" contains info about source element.
+    */
```

Comment:
When you have simple comments like this, just use `//` please.

```suggestion
        // "container" contains info about source element.
```

Reply:
Still pending.